cmd/cdi-controller/leaderelection.go

```diff
@@ -54,14 +66,17 @@ func startLeaderElection(ctx context.Context, config *rest.Config, leaderFunc fu
 	return nil
 }
 
-func createConfigMap(client kubernetes.Interface, namespace, name string) error {
+func createConfigMap(client kubernetes.Interface, cr *cdiv1.CDI, namespace, name string) error {
+	dynamicLabels := util.GetRecommendedLabels(cr, "cdi-controller")
```

Comment:
If we go this route, I think it would be better to have a function called `AddDynamicLabels` that takes a `metav1.Object`

Reply:
I guess if we go with env vars then this comment does not apply (correct me if I'm wrong?)

Just to understand, the suggestion here is to edit the cm after creation?